ansible/roles/tldraw-client-core/templates/tldraw-client-ingress.yml.j2

```diff
@@ -0,0 +1,49 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ NAMESPACE }}-tldraw-client-ingress
+  namespace: {{ NAMESPACE }}
+  annotations:
+    nginx.ingress.kubernetes.io/ssl-redirect: "{{ TLS_ENABLED | default("false") }}"
+    nginx.ingress.kubernetes.io/auth-type: basic
+    nginx.ingress.kubernetes.io/auth-secret: shd-basic-auth-secret
+    nginx.ingress.kubernetes.io/auth-realm: 'Authentication Required'
+{% if CLUSTER_ISSUER is defined %}
+    cert-manager.io/cluster-issuer: {{ CLUSTER_ISSUER }}
+{% endif %}
+
+spec:
+  ingressClassName: nginx
+{% if CLUSTER_ISSUER is defined or (TLS_ENABLED is defined and TLS_ENABLED | bool) %}
+  tls:
+  - hosts:
+      - {{ DOMAIN }}
+{% if CLUSTER_ISSUER is defined %}
+    secretName: {{ DOMAIN }}-tls
+{% endif %}
+{% endif %}
+  rules:
+  - host: {{ DOMAIN }}
+    http:
+      paths:
+      - path: /tldraw
+        pathType: Prefix
+        backend:
+          service:
+            name: tldraw-client-svc
+            port:
+              number: {{ PORT_TLDRAW_CLIENT }}
+      - path: /tldraw-client-runtime.config.json
+        pathType: Prefix
+        backend:
+          service:
+            name: tldraw-client-svc
+            port:
+              number: {{ PORT_TLDRAW_CLIENT }}
```

Comment:
this Ingress rule for the path `/tldraw-client-runtime.config.json` is not needed.
Why? We uns the pathType: Prefix
That mean if the ingress Rule for `/tldraw` matche an path is go to the tldraw-clinet-svc and in the end to the pod.
The path _**/tldraw**-client-runtime.config.json_ is includet in **/tldraw**

Other case where
```
      - path: /tldraw/
        pathType: Prefix
        backend:
          service:
            name: tldraw-client-svc
            port:
              number: {{ PORT_TLDRAW_CLIENT }}
```
Then the ingress rule woud only match all belwo the path `/tldraw/`